ftl/common/builder.py

```diff
@@ -73,17 +73,21 @@ def CreatePackageBase(self, base_image, cache):
     def BuildAppLayer(self):
         """Override."""
         buf = cStringIO.StringIO()
+        logging.info('Starting to generate tarfile from context...')
         with tarfile.open(fileobj=buf, mode='w') as out:
             for name in self._ctx.ListFiles():
                 content = self._ctx.GetFile(name)
                 info = tarfile.TarInfo(os.path.join('app', name))
                 info.size = len(content)
                 out.addfile(info, fileobj=cStringIO.StringIO(content))
+        logging.info('Finished generating tarfile from context.')
 
         tar = buf.getvalue()
         sha = 'sha256:' + hashlib.sha256(tar).hexdigest()
 
         gz = cStringIO.StringIO()
+        logging.info('Starting to gzip tarfile...')
```

Comment:
Do these include timestamps by default when printed, or do we need to customize the formatter?

Reply:
no it didn't.  I just updated the logging configuration to have millisecond timings so now it does.